docs/architecture/tech-stack.md

```
@@ -0,0 +1,366 @@
+
+| Technology            | Version  | Verwendung        | Begründung            |
+|-----------------------|----------|-------------------|-----------------------|
+| **OpenAPI Generator** | 7.x      | Client Generation | Type-safe API clients |
+| **TypeScript Axios**  | Template | Client Template   | Axios-based clients   |
+
+### Utilities
+
+| Technology   | Version | Verwendung         | Begründung                |
+|--------------|---------|--------------------|---------------------------|
+| **date-fns** | 4.x     | Date Manipulation  | Modular, tree-shakeable   |
+| **nanoid**   | 5.x     | ID Generation      | URL-safe unique IDs       |
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)